documentation/Overview.md

```diff
@@ -0,0 +1,88 @@
+<!--
+Copyright (c) 2019, NVIDIA CORPORATION.  All rights reserved.
+-->
+
+# Overview of Compiler Phases
+
+Each phase produces correct output or fatal errors.
+
+## Prescan and Preprocess
+
+**Input:** Fortran source and header files, command line macro definitions
```

Comment:
Also input: a set of enabled compiler directives (e.g., `OMP$`) that indicate which directives should be prescanned as part of the program instead of being treated like any other comments.